packages/flutter/lib/src/widgets/shortcuts.dart

```diff
@@ -386,6 +386,44 @@ class ShortcutManager extends ChangeNotifier with Diagnosticable {
 /// when invoking an [Action] via a keyboard key combination that maps to an
 /// [Intent].
 ///
+/// {@tool dartpad --template=stateful_widget_material}
+/// ```dart imports
+/// import 'package:flutter/services.dart';
+/// ```
+///
+/// ```dart preamble
+/// class Increment extends Intent {}
+///
+/// class Decrement extends Intent {}
+/// ```
+///
+/// ```dart
+/// int count = 0;
+///
+/// Widget build(BuildContext context) {
+///   return Shortcuts(
+///     shortcuts: <LogicalKeySet, Intent> {
+///       LogicalKeySet(LogicalKeyboardKey.alt, LogicalKeyboardKey.keyI): Increment(),
+///       LogicalKeySet(LogicalKeyboardKey.alt, LogicalKeyboardKey.keyD): Decrement(),
+///     },
+///     child: Actions(
+///       actions: <Type, Action<Intent>> {
+///         Increment: CallbackAction<Increment>(
+///          onInvoke: (Increment intent) => setState(()=> count = count + 1),
+///         ),
+///         Decrement: CallbackAction<Decrement>(
+///          onInvoke: (Decrement intent) => setState(()=> count = count - 1),
+///         ),
+///       },
+///       child: Center(
```

Comment:
We should provide some visual text here to recommend to users what they need to do to understand how this sample works. I'd recommend using a Row here that contains some text like:
```dart
Text('Add to the counter by pressing the keyboard numberpad "+" or Shift + "="')
Text('Decrement from the counter by pressing "-"')
```